nautobot/core/views/__init__.py

```diff
@@ -220,3 +225,40 @@ def render_graphiql(self, request, **data):
         data["saved_graphiql_queries"] = GraphQLQuery.objects.all()
         data["form"] = GraphQLQueryForm
         return render(request, self.graphiql_template, data)
+
+
+class NautobotAppMetricsCollector(Collector):
+    """Custom Nautobot metrics collector.
+
+    Metric collector that reads from registry["plugin_metrics"] and yields any metrics registered there."""
+
+    def collect(self):
+        """Collect metrics from plugins."""
+        start = time.time()
+        for metric in registry["app_metrics"]:
+            yield from metric()
+        gauge = GaugeMetricFamily("nautobot_app_metrics_processing_ms", "Time in ms to generate the app metrics")
+        duration = time.time() - start
+        gauge.add_metric([], format(duration * 1000, ".5f"))
+        yield gauge
+
+
+def custom_app_metric_view(request):
+    """Exports /metrics.
+
+    This overwrites the default django_prometheus view to inject metrics from Nautobot apps.
+
+    Note that we cannot use `prometheus_django.ExportToDjangoView`, as that is a simple function, and we need access to
+    the `prometheus_registry` variable that is defined inside of it."""
+    if "PROMETHEUS_MULTIPROC_DIR" in os.environ or "prometheus_multiproc_dir" in os.environ:
```

Comment:
Since the lower-case spelling didn't originate in this PR, should I open a separate issue that addresses this?

Reply:
I think this is more for us to keep in mind since we are mimicking their implementation. _If_ on the off-chance someone is still using it since it's still supported I think we shouldn't drop it pre-emptively, especially not in a patch release.